modules/core/04-channel/keeper/multihop_timeout_test.go

```diff
@@ -74,7 +74,10 @@ func (suite *MultihopTestSuite) TestTimeoutPacket() {
 					// proof of absence of packet receipt
 					key = host.PacketReceiptKey(packet.SourcePort, packet.SourceChannel, packet.Sequence)
 				}
-				proof = suite.Z().QueryMultihopProof(key, packetHeight)
+				var err error
```

Comment:
i think this is not needed since there is an outer error var?

Reply:
Fixed